### PR TITLE
meson: update to 0.53.1

### PR DESCRIPTION
**Pull request by teldra**

### CODE REVIEW

File: srcpkgs/meson/template [hunk 1]
```diff
@@ -1,11 +1,10 @@
 # Template file for 'meson'
 pkgname=meson
 reverts="0.52.0_1 0.52.0_2"
```
Comment (sgn) [inline srcpkgs/meson/template]: Remove `reverts`

File: srcpkgs/meson/template [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
-version=0.52.1
-revision=2
+version=0.53.1
+revision=1
 archs=noarch
 build_style=python3-module
-pycompile_module="mesonbuild"
 hostmakedepends="python3-devel python3-setuptools"
 depends="ninja python3-setuptools"
 checkdepends="glib-devel gobject-introspection libsanitizer-devel ninja
```
Comment (sgn) [inline srcpkgs/meson/template]: `libsanitizer-devel` isn't available on musl

Comment (teldra) [reply srcpkgs/meson/template]: What is the best way to manage this?

Comment (sgn) [reply srcpkgs/meson/template]: Something like:

```sh
if [ "$XBPS_TARGET_LIBC" = glibc ]; then
	checkdepends+=" libsanitizer-devel"
fi
```

should work.

IIRC, some tests failed badly without libsanitizer.

File: srcpkgs/meson/template (hunk 1 above)
Comment (teldra) [reply srcpkgs/meson/template]: done

File: srcpkgs/meson/template (hunk 2 above)
Comment (teldra) [reply srcpkgs/meson/template]: done

**Comment by teldra** [comment]
ah. maybe this pr is over my head.
xbps-src check do not run through without errors on x8_64.

**Comment by sgn** [comment]
On 2020-02-05 23:55:00-0800, Teldra <notifications@github.com> wrote:
> ah. maybe this pr is over my head.
> xbps-src check do not run through without errors on x8_64.

The old comment said there were 3 test cases failed in the old days.

meson requires a trillion dependencies to run the test.

-- 
Danh

**Pull request closed without merging**